### DISCUSSION TITLE
哥布林不要幻想

### DISCUSSION THREAD

**Discussion by Bahajsjsh** [share121/inter-knot#2724] [其他]
![emotion_download_1722270219835](https://github.com/user-attachments/assets/10d68b0a-5ec7-43a0-87a4-18bc2efa92a5)

**Comment by demolute** [comment]
请问是木叶村的一打七先生吗？

**Comment by OttoApocalypse01** [comment]
我猜你想发幻影坦克但是失败了<img src="https://bbs-static.miyoushe.com/static/2024/07/31/213614cd6275ec54d80fe9f84a0edf62_6334766189957723007.png" alt="白露 灭火" width="75" >
